server/models/abstract.js

```diff
@@ -132,6 +165,38 @@ const Abstract = db.bookshelf.Model.extend({
 
     return target;
   }
+}, {
+
+  /**
+   * Returns a query builder initialized with this model as a base query.
+   *
+   * @method
+   * @memberof Abstract
+   * @static
+   * @param {object} options - The query builder's options.
+   * @returns {OrmQueryBuilder} A query builder.
+   *
+   * @see {@link https://github.com/MediaComem/orm-query-builder}
+   */
+  queryBuilder: function(options) {
```

Comment:
This method has already been defined at the top of the file.
All the code that comes after looks like a duplicate from the one at the top.